_data/toc.yaml

```
@@ -1274,13 +1274,11 @@ manuals:
         - path: /desktop/dev-environments/
           title: Overview
         - path: /desktop/dev-environments/create-dev-env/
-          title: Create a Dev Environment
+          title: Create a simple dev environment
```

I think we can cut out simple, WDYT? 

```suggestion
          title: Create a dev environment
```